packages/scaffold-config/src/index.ts

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable padding-line-between-statements */
```

Comment:
The file above this is "Load diff" but you should look at it, that's where all the frameworks and their configuration is.